config/routes.rb

```diff
@@ -1,60 +1,14 @@
 Rails.application.routes.draw do
 
   resources :companies do
-    resources :jobs
+    resources :jobs do
+      resources :comments, only: [:create]
```

Comment:
we talked about this a bit in class (about nesting three deep), let me know if you want to talk through this at all again :thumbsup: